### DISCUSSION TITLE
I still have an early nightly version installed

### DISCUSSION THREAD

**Discussion by Wiw1** [ruffle-rs/ruffle#14500] [Q&A]
I installed the Firefox extension over a previous nightly build (23-8-2), but when I remove or disable that version, the previous build still runs. There's no way I see of disabling this one. Is there anything I can do about it?

**Comment by n0samu** [comment]
It sounds like the website you are visiting probably has an old version of Ruffle installed. See our FAQ: https://github.com/ruffle-rs/ruffle/wiki/Frequently-Asked-Questions-For-Users#how-do-i-uninstall-ruffle

**Comment by Wiw1** [comment]
No, it definitely isn't that. I was looking at a page in the Wayback Machine.

**Reply by n0samu** [reply]
The Wayback Machine actually does have Ruffle installed. They use it on archived pages with Flash content to make them work in modern browsers. If you use a browser with the real Flash Player installed, it should fall back to that instead of using Ruffle.

**Comment by Wiw1** [comment]
Oh, I understand now.
